### PR TITLE
Added source acknowledgement

### PR DESCRIPTION
**Pull request by jbeemster**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 A Terraform module which deploys the requisite micro-services for loading BigQuery on Google running on top of Compute Engine.  If you want to use a custom image for this deployment you will need to ensure it is based on top of Ubuntu 20.04.
 
+## Source acknowledgement
+
+This module was originally sourced from a [community contribution](https://github.com/tnightengale/snowplow-terraform-google-big-query-loader-ce/tree/main?tab=readme-ov-file) by [Teghan Nightengale](https://github.com/tnightengale) in 2022 - big thanks for the help in getting this one started!
```
Comment (tnightengale) [inline README.md:9]:
```suggestion
This module was originally sourced from a [community contribution](https://github.com/tnightengale/snowplow-terraform-google-big-query-loader-ce/tree/main?tab=readme-ov-file) by [Teghan Nightengale](https://github.com/tnightengale) in 2022 - big thanks for the help in getting this one started!
```

**Review by tnightengale — CHANGES_REQUESTED** [review]
@jbeemster Would you mind accepting this proposed change, so I can formally add a commit to the project and appear as a contributor?

File: README.md (hunk 1 above)
Comment (jbeemster) [inline README.md:9]: Hey @tnightengale I can't accept the commit without there being an actual change to commit.

File: CHANGELOG [hunk 2]
```diff
@@ -1,6 +1,10 @@
+Version 0.3.2 (2024-08-21)
+--------------------------
+Credit Original Author (#11)
```
Comment (tnightengale) [inline CHANGELOG]: @jbeemster added a new suggestion here, and deleted the other non-changing commit suggestion.
```suggestion
Credit Original Author, Teghan Nightengale (#11)
```

**Pull request merged**